runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -346,7 +343,7 @@ public void init(final RuneLite runelite) throws Exception
 
 			// Add key listener
 			final HotkeyListener sidebarListener = new HotkeyListener(() ->
```

Comment:
This can be shortened to `final HotkeyListener sidebarListener = new HotkeyListener(config::sidebarToggleKey);`